src/LazyBulderCommentsStatistics.php

```diff
@@ -12,7 +13,7 @@ class LazyBulderCommentsStatistics {
   /**
    * Drupal\statistics\NodeStatisticsDatabaseStorage definition.
    *
-   * @var \Drupal\Core\Entity\EntityManagerInterface
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
    */
   protected $entityManager;
```

Comment:
We can also change the name of the variable to `entityTypeManager`. But that's not mandatory.

Reply:
Updated. Thanks for pointing out